configs/r21d.yml

```diff
@@ -1,7 +1,8 @@
 # Model
 feature_type: 'r21d'
-stack_size: 16 # Feature time span in fps
-step_size: 16 # Feature step size in fps
+model_name: 'default'
```

Comment:
Ok, but can we name it more informatively? E.g. `default` --> `r2plus1d_18_16_kinetics`